src/lucky_record/charms/string_extensions.cr

```diff
@@ -15,14 +15,12 @@ class String
       @upper = false
       @lower = false
 
-      def like(value : String)
-        rows.query.where(LuckyRecord::Where::Like.new(column, value))
-        rows
+      def like(value : String) : T
```

Comment:
A test for the negation of these might be good. Maybe in `spec/string_critiera_spec.cr

Reply:
These tests already exist. See `spec/string_criteria_spec.cr` exactly 🍾

Reply:
Oops! I should look more closely! :D